stl/inc/experimental/generator

```diff
@@ -55,19 +56,29 @@ namespace experimental {
                 return {};
             }
 
-#ifdef _CPPUNWIND
+#if defined(_CPPUNWIND) || defined(__cpp_impl_coroutine)
             void unhandled_exception() noexcept {
+#if 1 // TRANSITION, VSO-1172852
+#ifdef _CPPUNWIND
                 _Eptr = _STD current_exception();
-            }
+#else // ^^^ _CPPUNWIND / !_CPPUNWIND vvv
+                abort();
 #endif // _CPPUNWIND
+#else // ^^^ workaround / no workaround

```

Comment:
Are you missing `vvv` here? 😸 (Ditto on line 133.)

Reply:
```suggestion
#else // ^^^ workaround / no workaround vvv
```